docs/geoQueryService.md

```diff
@@ -0,0 +1,88 @@
+# GeoQueryService API Documentation
```

Comment:
I am thinking we should use the term "geoQueryService" to match the name of the REST extension. 

This introduction here should also make it clear what the API actually is - i.e. it's the "/v1/resources/geoQueryService" API.

Reply:
I would recommend starting with a technical API description in the shape of an OpenAPI spec. There is room for descriptions about nitty grittty details in there, and then add a higher level description in this MD, with a ref to the spec.